app/Console/MarkPlaceAsDuplicateInBulkCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use Broadway\CommandHandling\CommandBus;
+use Broadway\EventHandling\EventListener;
+use CultuurNet\UDB3\Place\CannotMarkPlaceAsCanonical;
+use CultuurNet\UDB3\Place\CannotMarkPlaceAsDuplicate;
+use CultuurNet\UDB3\Place\Canonical\CanonicalService;
+use CultuurNet\UDB3\Place\Canonical\DuplicatePlaceRepository;
+use CultuurNet\UDB3\Place\Commands\MarkAsDuplicate;
+use Psr\Log\LoggerAwareInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class MarkPlaceAsDuplicateInBulkCommand extends AbstractCommand
```

Comment:
I would maybe change the name to something like `MarkPlacesAsDuplicate` with a good suffix indicating the duplicate places table.